releasetasks/templates/firefox/partner_repacks.yml.tmpl

```diff
@@ -42,6 +43,50 @@
 
         extra:
             {{ common_extras(taskname=buildername, locales=["null"], platform="null") | indent(12)}}
+
+-
+    taskId: "{{ stableSlugId(buildername_eme_free) }}"
+    reruns: 5
+    requires:
+        {% for upstream_builder in artifact_completes_builders %}
+        - {{ stableSlugId(upstream_builder) }}
+        {% endfor %}
+    task:
+        provisionerId: "buildbot-bridge"
+        workerType: "buildbot-bridge"
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+            - project:releng:buildbot-bridge:builder-name:{{ buildername }}
+        routes:
+            - index.releases.v1.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.eme_free_repacks.{{ platform }}
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.eme_free_repacks.{{ platform }}
+        payload:
+            buildername: "{{ buildername }}"
+            sourcestamp:
+                branch: "{{ repo_path }}"
+                revision: "{{ mozharness_changeset }}"
+            properties:
+                product: "{{ product }}"
+                version: "{{ version }}"
+                build_number: {{ buildNumber }}
+                repo_path: "{{ repo_path }}"
+                script_repo_revision: "{{ mozharness_changeset }}"
+                release_promotion: true
+                revision: "{{ mozharness_changeset }}"
+                repack_manifests_url: "https://github.com/mozilla-partners/mozilla-EME-free-manifest"
```

Comment:
The first one is a private repo, the second one is public. I could use git:// for the second one, but https:// requires no secrets.